Dockerfile.ocp

```diff
@@ -13,7 +13,7 @@ RUN dnf upgrade -y && \
       && \
     dnf download --destdir /tmp/packages \
       openstack-ironic-python-agent \
-      python3-bcrypt \
+      python3-bcrypt.x86_64 \
```

Comment:
Do we know how OCP multi-arch builds work?  I assume some variable is passed in at build time, so I wonder can we find an example of that and do the same here to avoid hard-coding?

Reply:
Or I see we're using $(uname -m) to determine the arch for the IPA RPM, can we do the same here?

Reply:
We need to hard-code here because we're installing in ipa-ramdisk which is x86-64 only
Unfortunately in the images both x86-64 and ppc64 repositories are included, which is ok if we need to control only what's installed on the image itself.
In case we need to do something exotic (like this case or like the builder in the ironic image), we have to force the arch.

Reply:
scratch that, I forgot we also build (and install) ipa-ramdisk natively on ppc64, so we need to change arch to variable, not hard-code it
thanks for that!